lib/utils.js

```diff
@@ -677,7 +675,7 @@ exports.stackTraceFilter = function() {
       }
 
       // Clean up cwd(absolute)
-      if (/\(?.+:\d+:\d+\)?$/.test(line)) {
+      if (/:\d+:\d+\)?$/.test(line)) {
         line = line.replace('(' + cwd, '(');
```

Comment:
What is the purpose of this change?

Reply:
as explained in the description of this pr, the original regex performs poorly if `line` is larger than 20k, which is quite possible if assertions on large string were made. The modified regex checks the same thing without performance issue.

The removed part in the original regex, `/\?.+/`, basically matches any character but it's actually unnecessary because we only to find lines with line and col numbers separated by `:` which is already written out.

Reply:
Is there a timing test where this speedup is shown?
```sh
$ time mocha test.spec.js
$ time mocha-patched test.spec.js
```

Reply:
> as explained in the description of this pr, the original regex performs poorly if `line` is larger than 20k,
> which is quite possible if assertions on large string were made. The modified regex checks the same thing 
> without performance issue.

If the `Error.message` is causing the problem, why not just regex test `Error.stacktrace.slice(1)`?

Reply:
I think you mean `stack`, but it's not that simple: it's a string, and the message may be multi-line.

Reply:
regarding benchmarks, we have none, and I'd be happy to accept a PR with harnesses/scripts/dependencies to make something work... but it doesn't need to be in this PR.

Reply:
> I think you mean `stack`, but it's not that simple: it's a string, and the message may be multi-line.

Fair enough. I was just talking basic idea, but now see we only have the stacktrace, not the `Error` itself.